### PR TITLE
Data page

### PR DESCRIPTION
**Pull request by XinlongH6**
Might need to reset the database and reset to test the charts. Added stuff to the seed to add a years worth of orders (2 per day)

When logging as an admin, it redirects you to the data page immediately: might need to fix

Bar chart gets data from prisma, Sunburst does not currently (hardcoded)

### CODE REVIEW

File: pages/data.vue [hunk 1]
```diff
@@ -1,17 +1,17 @@
 <template lang="pug">
-  #app
-  div.mx-auto(class="w-3/4")       
-    //1st chart
-    PlotlyChart(:data="chartData" :layout="chartLayout" :config="chartConfig")
+  div
+    #app
```
Comment (emoryjgrubbs) [inline pages/data.vue:3]: you don't need the #app

**Review by emoryjgrubbs — COMMENTED** [review]
charts, look good

i'm getting an error when refreshing the data page (may be an issue with plotly?)
image
image

you'll need to take a look at the styling for mobile, specifically the bar chart (it's not a priority, but we should still try to support it)
image

i'm not seeing much you can do, but the chart buttons are misaligned with the tool tips on firefox (make an issue and it can be looked into later)
image

File: pages/data.vue (hunk 1 above)
Comment (XinlongH6) [reply pages/data.vue:3]: Not sure what the error is due to, Daniel is going to ask Taz

Formatting for mobile should be solved when aggregating the data based on month/year, also removed 3/4

Removed the tool tips w/ displayModeBar: false on both charts
Might be removing the sunburst chart to streamline a chart system where you can generate charts based on Daniels criterias and then send to plotly to generate the chart

Time level: Hour, day, week, month, semester, time range
Time filters: time range to look at
View level: Inventory, per item, per category, per source
View filters: multiselect items, categories, or sources

removed #app

**Comment by emoryjgrubbs** [comment]
i think daniel should be included in how exactly we want to handle the error
my main view is that we should make an issue and merge this, the changes look good and i don't wan to have everything in one giant pr

File: package.json [hunk 2]
```diff
@@ -24,13 +24,16 @@
 		"@prisma/client": "^5.16.2",
 		"nanoid": "^5.0.7",
 		"nuxt": "^3.12.2",
+		"plotly.js-dist": "^2.35.2",
```
Comment (CrazEpic) [inline package.json:27]: figure out exactly which one we need and delete the other

**Review by CrazEpic — APPROVED** [review]
decent

**Pull request merged**